src/components/Groups.tsx

```diff
@@ -0,0 +1,25 @@
+import Button from './Button'
+
+interface Props {
```

Comment:
Maybe a more specific name? 
```suggestion
interface GroupsProps {
```

Reply:
My idea was to keep it short and sweet since the interface is not exported. In that case I would have kept the more specific name.